domains/zoom.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "A way of showing my github website",
+    "repo": "https://github.com/theoldzoom/theoldzoom.github.io",
+    "owner": {
+        "username": "theoldzoom",
+        "email": "theoldzoom@gmail.com",
+        "twitter": "theoldzoom",
+        "discord": "theoldzoom",
```

Comment:
remove the comma at the "discord" line

Reply:
I just did. Thanks you for telling me about this mistake I've made.